g1_map_robot.cydsn/ZumoLibrary/Lidar.c

```diff
@@ -0,0 +1,39 @@
+#include "Lidar.h"
+#include "stdint.h"
+
+static int old_dist = -1;
+
+void
+Lidar_start()
+{
+  UART_3_Start();
+  Lidar_get_distance();
+}
+
+int
```

Comment:
Can we name all the current constants? Hard to understand why these are particularly certain numbers 🧐

Reply:
Agree, in Lidar.h we can have macros for known constants, for example:
```#define UART_PKG_SIZE 9
#define UART_PKG_HEADER 0x59
```